termbox/src/lib.rs

```diff
@@ -405,6 +383,57 @@ impl Termbox {
             .unwrap();
         }
     }
+
+    fn send_char(&mut self, to_x: u16, to_y: u16, ch: char) {
+        // if the target cell is next to the last cell, then don't move the cursor
+        if (self.last_cursor.0 + 1) == to_x && self.last_cursor.1 == to_y {
+            write!(self.output_buffer, "{}", ch).unwrap();
+        } else {
+            goto(to_x, to_y, &mut self.output_buffer);
+            write!(self.output_buffer, "{}", ch).unwrap();
+        }
+        self.last_cursor = (to_x, to_y);
+    }
+}
+
+const LOOKUP: &[u8] = b"0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+fn num_to_buf(mut num: u16, buf: &mut Vec<u8>) {
+    let start_len = buf.len();
+    let mut chars_len = 0;
+    loop {
+        let rem = num % 10;
+        let ch = LOOKUP[rem as usize];
```

Comment:
This could be `let ch = rem as u8 + b'0';`

Reply:
Agreed, let's update this.

Reply:
Agreed.